### PR TITLE
New/view workspace users

### PR DESCRIPTION
**Pull request by ursarik**

### CODE REVIEW

File: frontend/src/services/workspace-api/workspace-api.service.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { HttpMethod, ContentType } from 'common/enums/enums';
+import { Http } from 'services/http/http.service';
+
+const http = new Http();
+
+class WorkspaceApi {
+  private BASE = '/api/workspaces/';
+
+  public async loadUsers(): Promise<IWorkspaceUser[]> {
+    const response: IWorkspaceUser[] = await http.load(this.BASE + 'users', {
```
Comment (aleksandr-kuharenko) [inline frontend/src/services/workspace-api/workspace-api.service.ts:11]:
```suggestion
  private BASE = '/api/workspaces';

  public async loadUsers(): Promise<IWorkspaceUser[]> {
    const response: IWorkspaceUser[] = await http.load(`${this.BASE}/users`, {
```

File: backend/src/services/workspace.service.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { getCustomRepository } from 'typeorm';
+import { IWorkspaceUser } from '../common/interfaces/workspace/workspace-user';
+import { mapWorkspaceToWorkspaceUsers } from '../common/mappers/workspace/map-workspace-to-workspace-users';
+import WorkspaceRepository from '../data/repositories/workspace.repository';
+
+export const getUsers = async (
```
Comment (aleksandr-kuharenko) [inline backend/src/services/workspace.service.ts:6]:
```suggestion
export const getWorkspaceUsers = async (
```

File: frontend/src/components/workspace/users-settings/user-item/teams/teams.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { IWorkspaceUser } from '../../../../../common/interfaces/workspace/index';
```
Comment (AlexKapic) [inline frontend/src/components/workspace/users-settings/user-item/teams/teams.tsx:1]: maybe import from 'infostack-shared/common/interfaces'

File: frontend/src/components/workspace/users-settings/user-item/add-role/add-role.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import styles from '../../styles.module.scss';
+
+const AddRole: React.FC = () => {
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/workspace/users-settings/user-item/add-role/add-role.tsx:3]: I think, "AddTeam" will be better
```suggestion
const AddTeam: React.FC = () => {
```

File: frontend/src/services/workspace-api/workspace-api.service.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      method: HttpMethod.GET,
```
Comment (AlexKapic) [inline frontend/src/services/workspace-api/workspace-api.service.ts:12]: in http.service GET it's default method

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

File: frontend/src/components/workspace/users-settings/user-item/teams/teams.tsx (hunk 3 above)
Comment (aleksandr-kuharenko) [reply frontend/src/components/workspace/users-settings/user-item/teams/teams.tsx:1]:
```suggestion
import { IWorkspaceUser } from 'common/interfaces/workspace/index';
```

File: frontend/src/components/workspace/users-settings/user-item/add-role/add-role.tsx (hunk 4 above)
Comment (ursarik) [reply frontend/src/components/workspace/users-settings/user-item/add-role/add-role.tsx:3]: Dunno why i decided to name it AddRole :confused:

File: backend/src/api/routes/workspace-route.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+import { getWorkspaceUsers } from '../../services/workspace.service';
+
+const router: Router = Router();
+
+// TODO: add security middleware's
+router.get(
+  '/users',
+  // TODO: change with req.cookies.workspaceId
+  run(() => getWorkspaceUsers('b6e959fd-09b3-42cd-8a30-90c31054198a')),
+);
```
Comment (aleksandr-kuharenko) [inline backend/src/api/routes/workspace-route.ts:12]: PR with middlewares merged, you can update this

File: backend/src/common/mappers/workspace/map-workspace-to-workspace-users.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+export const mapWorkspaceToWorkspaceUsers = (
+  workspace: Workspace,
+): IWorkspaceUser[] => {
+  return workspace.userWorkspaces.map(({ user, role }) => {
+    return {
+      id: user.id,
+      fullName: user.fullName,
+      role,
+      teams: user.teams.map((team) => team.name),
```
Comment (aleksandr-kuharenko) [inline backend/src/common/mappers/workspace/map-workspace-to-workspace-users.ts:12]:
```suggestion
    const { id, fullName, teams } = user;
    return {
      id,
      fullName,
      role,
      teams: teams.map(team => team.name),
```

File: frontend/src/components/workspace/users-settings/user-item/user-item.tsx [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import { getAllowedClasses } from '../../../../helpers/dom/dom';
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/workspace/users-settings/user-item/user-item.tsx:1]:
```suggestion
import { getAllowedClasses } from 'helpers/dom/dom';
```

File: frontend/src/components/workspace/users-settings/user-item/user-item.tsx [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+import styles from '../styles.module.scss';
+import { IWorkspaceUser } from '../../../../common/interfaces/workspace';
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/workspace/users-settings/user-item/user-item.tsx:3]: Update all nested imports 
```suggestion
import { IWorkspaceUser } from 'common/interfaces/workspace';
```

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by AlexKapic — APPROVED** [review]

**Pull request merged**